doc/neps/nep-0041-improved-dtype-support.rst

```diff
@@ -0,0 +1,706 @@
+=================================================
+NEP 41 — First step towards a new Datatype System
+=================================================
+
+:title: Improved Datatype Support
+:Author: Sebastian Berg
+:Author: Add...
+:Status: Draft
+:Type: Standard Track
+:Created: 2020-02-03
+
+
+.. note::
+
+    This NEP is part of a series of NEPs encompassing first information
+    about the previous dtype implementation and issues with it in NEP 40.
+    NEP 41 (this document) then provides an overview and generic design
```

Comment:
Can we make these links?

Reply:
Yes, but only as they are being merged, and I hope to merge this one first.